scripts/build/Platform/Windows/build_config.json

```diff
@@ -36,6 +36,19 @@
       "test_cpu_profile_vs2019"
     ]
   },
+  "aws_integration_deployment": {
+    "TAGS": [
+      "nightly-aws-integration-deployment",
+      "weekly-build-metrics"
```

Comment:
Does this need to run in weekly-build-metrics?

Reply:
I thought the tag was for collecting metrics instead of an actual Jenkins job. Will remove it then. Thanks.